config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -1246,7 +1232,7 @@ def generate_presubmits_e2e():
             scenario='upgrade-ab',
             env={
                 'KOPS_VERSION_A': "1.23",
-                'K8S_VERSION_A': "v1.25.0",
+                'K8S_VERSION_A': "v1.23.0",
```

Comment:
I have doubts this can be correct, why kops will want to go back instead of test the latest

Reply:
becuase of this 1 unit test that keeps failing... I wanted to see if this merge was the issue